test/test_emulation/__init__.py

```diff
@@ -92,12 +92,19 @@ def udev_rules(self):
     def server(self):
         p = None
         try:
-            p = subprocess.Popen('python -m ratbag_emu 2&>1 >/dev/null', shell=True)
+            stdout = open('ratbag-emu-log-stdout.txt', 'w')
+            stderr = open('ratbag-emu-log-stderr.txt', 'w')
+
+            p = subprocess.Popen(['/usr/bin/env', 'python3', '-m', 'ratbag_emu'], stdout=stdout, stderr=stderr)
```

Comment:
I think you may be able to hook up a logger for this too and then pass the logger through as stdout/stderr. Something to put onto the nice-to-have list.